clusters/gitlab-cluster/gitlab/gitlab.yaml

```diff
@@ -61,6 +61,9 @@ spec:
             service.beta.kubernetes.io/aws-load-balancer-backend-protocol: "tcp"
             service.beta.kubernetes.io/aws-load-balancer-ssl-ports: "443"
             service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
```

Comment:
This should be deleted. It's redefined below.

Reply:
Also "proxy-protocol" was latent until this change, but it enables v2, which looks like breaks us. Please delete that line above.